src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -24,14 +26,50 @@
  *
  * @author Kuanhung Chen <ericj.tw@gmail.com>
  */
-final class MethodArgumentSpaceFixer extends AbstractFixer
+final class MethodArgumentSpaceFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
     /**
+     * @var array<string, bool>
+     */
+    private static $defaultConfiguration = array('keepMultipleSpacesAfterComma' => false);
+
+    /**
+     * Preserve existing multiple spaces after comma.
+     *
+     * @var bool
+     */
+    private $keepMultipleSpacesAfterComma;
+
+    /**
+     * @param null|array $configuration
+     *
+     * @throws InvalidFixerConfigurationException
+     */
+    public function configure(array $configuration = null)
+    {
+        if (null === $configuration) {
+            $configuration = self::$defaultConfiguration;
+        }
+
+        if (!array_key_exists('keepMultipleSpacesAfterComma', $configuration)) {
+            throw new InvalidFixerConfigurationException($this->getName(), 'Missing "keepMultipleSpacesAfterComma" configuration.');
```

Comment:
i don't like making this key obligatory.
then, we will introduce 2nd key, `foo`.
then, I want to be able to only use `foo` key and keep this one with default value.